newIDE/app/src/ObjectEditor/ObjectEditorDialog.js

```diff
@@ -68,6 +75,18 @@ export class ObjectEditorDialog extends Component<*, StateType> {
         }
         titleStyle={styles.titleContainer}
       >
+      Object Name:
+      <SemiControlledTextField
+          style={styles.objectTitle}
+          commitOnBlur
```

Comment:
Can you try to pass "fullWidth" prop?

Reply:
The fullWidth prop pushed it to another row, so instead I used a div container with css style inspired from another place in the editor